### DISCUSSION TITLE
validateStatus option not working.

### DISCUSSION THREAD

**Discussion by LuckyArdhika** [axios/axios#4881] [Q&A] [answered]
Im using NodeJS and Axios v0.27.02. I want to including 302 status code only for valid response and exclude other status code as an error, so im using validateStatus as decribed in the [documentation](https://axios-http.com/docs/handling_errors), but my code didnt working as expected.

My code:
```
return axios({
      url: url_Here,
      method: 'GET',
      withCredentials: true,
      headers: {
        'Content-Type': 'application/x-www-form-urlencoded',
        'User-Agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36",
        'cookie': cookiesParsed.toString().replace(",",";")
      },
      maxRedirects: 1,
      validateStatus: (status: number) => status == 302
    }).then(async function(response: any){
          console.log('OK');
    }).catch(err => debug(err))
```
I want to display OK, but the console show an error ERR_FR_TOO_MANY_REDIRECTS.
The API working fine in the http request tool with status code 302. How i can resolve this error? i want to get headers fron response.

**Comment by jasonsaayman** [comment, marked as answer]
302 is normally a redirect code, just for sanity I would suggest the API itself be changed else you are breaking a web standard.
